pkg/networkutils/network.go

```diff
@@ -745,36 +746,28 @@ func setupENINetwork(eniIP string, eniMAC string, eniTable int, eniSubnetCIDR st
 			return errors.Wrap(err, "setupENINetwork: failed to clean up old routes")
 		}
 
-		// In case of route dependency, retry few times
-		retry := 0
-		for {
+		err = retry.RetryNWithBackoff(retry.NewSimpleBackoff(time.Millisecond*500, retryRouteAddInterval*time.Duration(maxRetryRouteAdd), 0.15, 2.0), maxRetryRouteAdd, func() error {
```

Comment:
Nice! I do think we could reduce the max time though to just the `retryRouteAddInterval`, since that is 5s:
```
retry.NewSimpleBackoff(0.5 * time.Second, retryRouteAddInterval, 0.15, 2.0)
```
This would give the delays (not counting the jitter): 0.5, 1, 2, 4, 5 seconds.